src/rlx_prv_relup.erl

```diff
@@ -145,7 +155,14 @@ make_upfrom_script(State, Release, UpFrom) ->
                             "relup successfully created!"),
             {ok, State};
         {ok,_, Module,Warnings} ->
-            ?RLX_ERROR({relup_script_generation_warn, Module, Warnings});
+            case WarningsAsErrors of
```

Comment:
i did as you suggested, yet i wasn't expecting for `systools:make_relup` to return simply `error` when i give it the `warnings_as_errors` option instead of `{error,Module,Errors}`, intended behaviour?